src/depends/safeserver/safehttpserver.cpp

```diff
@@ -107,20 +114,20 @@ bool SafeHttpServer::StartListening() {
             SafeHttpServer::callback, this, MHD_OPTION_HTTPS_MEM_KEY,
             this->sslkey.c_str(), MHD_OPTION_HTTPS_MEM_CERT,
             this->sslcert.c_str(), MHD_OPTION_THREAD_POOL_SIZE, this->threads,
+            MHD_OPTION_LISTEN_BACKLOG_SIZE, LISTEN_BACKLOG_SIZE,
```

Comment:
Could you add a comment about these new options. Why are they needed here and what they do?